setup.py

```diff
@@ -42,7 +42,7 @@
     "scikit-hep>=0.5.0, <0.6.0",
     "six",
     "torch>=1.0.0",
-    "uproot",
+    "uproot>=3.11.0,<4.0.0",
```

Comment:
```suggestion
    "uproot3>=3.14.1",
```

Jim made `uproot3` on PyPI to help deal with these problems https://pypi.org/project/uproot3/#history